deltacat/compute/compactor/compaction_session.py

```diff
@@ -334,6 +335,14 @@ def _execute_compaction_round(
             "Multiple rounds are not supported. Please increase the cluster size and run again."
         )
 
+    cluster_resources_usage = get_current_cluster_resources_usage()
```

Comment:
For this common case, may want to consider another utility function that wraps `invoke_parallel`, includes the call to `ray.get()`, and has an argument like `log_utilization_diff` to add before/after log-lines that print out all dictionaries inside of the `ClusterUtilization` class before and after any requested parallel invocation completes.

Reply:
created `log_current_cluster_utilization` method.